internal/dataplane/translator/translator.go

```diff
@@ -120,6 +121,10 @@ type Config struct {
 
 	// ClusterDomain is the cluster domain used for translating Kubernetes objects.
 	ClusterDomain string
+
+	// GatewayControllerName is the gateway controller name used by KIC.
+	// GatewayClasses with this controller name in spec.ControllerName are managed by KIC, otherwise they are managed by other components(like Kong Operator).
```

Comment:
```suggestion
	// GatewayClasses with this controller name in spec.ControllerName are managed by KIC, otherwise they are managed by other components (like Kong Operator).
```